app/services/container_dashboard_service.rb

```diff
@@ -30,7 +30,7 @@ def status
 
     {
       :nodes      => {
-        :count        => @ems.present? ? @ems.container_nodes.count : ContainerNode.count,
+        :count        => @ems.present? ? @ems.container_nodes.count : ContainerNode.where.not(:ext_management_system => nil).count,
```

Comment:
Same, can you add scope to model and say `ContainerNode.active.count` here?
`:ext_management_system => nil` would stop working with switch to soft deletes.